### ISSUE TITLE
MODBUS - device-modbus-go

### ISSUE THREAD

**Issue by boaretti** [edgexfoundry/device-modbus-go#369]
Hi,
Please, when reading a single HOLDING_REGISTER is it possible to decouple a single read in multiple
values. My case a single register the information represented as BITs so If possible I would like to derive several information
from this single read. Is it possible ? Could someone point me to the correct place ?

**Comment by cloudxxx8** [comment]
Yes, it's possible by modifying the driver code.  You can feel free to open a PR for this mechanism.

**Comment by jiekechoo** [comment]
@cloudxxx8 
I have a modbus tcp slave, it has  >5 register data, is it possible to `couple` to one event in `AutoEvents`?

> Yes, it's possible by modifying the driver code. You can feel free to open a PR for this mechanism.

**Comment by cloudxxx8** [comment]
Yes, you define one Device Command in the Device Profile to aggregate multiple Device Resources (5 in your case).
https://docs.edgexfoundry.org/2.3/walk-through/Ch-WalkthroughDeviceProfile/#understanding-device-commands

**Comment by jiekechoo** [comment]
@cloudxxx8 
![image](https://user-images.githubusercontent.com/3350211/218398090-75c5569d-eecc-415f-bcbf-c9d1aaca65ac.png)
got it, thanks.
